### DISCUSSION TITLE
Change color of dataset

### DISCUSSION THREAD

**Discussion by tnguy271** [GSI-CS-CO/chart-fx#587] [Q&A]
Hi everyone,
I'm trying to change the color of a dataset to red. I use this code and was able to do it, except the marker line is still blue. Does anyone know how to resolve this? Here is my code and screenshot
```
dataSet.setStyle("strokeColor=#FF0000");
dataSet.setStyle("fillColor=#FF0000");
foregroundRenderer.getDatasets().add(dataSet);
```
where dataSet is DoubleDataSet and foregroundRenderer is ErrorDataSetRenderer. 
<img width="920" alt="Screenshot 2023-06-19 at 2 58 44 PM" src="https://github.com/fair-acc/chart-fx/assets/114690375/6e3b89fb-b333-4893-9b0b-db35cdc79986">

Thank you so much !

**Comment by tnguy271** [comment]
dataSet.setStyle("fillColor=#FF0000; markerColor=#FF0000; strokeColor=#FF0000;");

**Reply by wirew0rm** [reply]
Did this solve your problem or is this just something else you tried? If yes, please accept it as the correct answer, otherwise i'll try to look into it but that would require a bit more information, os, java and chartfx version and ideally a minimum viable example or a way to reproduce it by adapting one of our samples.
